charts/camunda-platform/templates/connectors/deployment.yaml

```diff
@@ -0,0 +1,93 @@
+        annotations: {{- toYaml  .Values.connectors.podAnnotations | nindent 8 }}
+        {{- end }}
+    spec:
+{{/*      imagePullSecrets:*/}}
+{{/*        {{- include "camundaPlatform.imagePullSecrets" . | nindent 8 }}*/}}
+      containers:
+        - name: connectors
+          image: {{ include "camundaPlatform.imageByParams" (dict "base" .Values.global "overlay" .Values.connectors) }}
+          imagePullPolicy: {{ .Values.global.image.pullPolicy }}
+          {{- if .Values.connectors.containerSecurityContext }}
+          securityContext: {{- toYaml .Values.connectors.containerSecurityContext | nindent 10 }}
+          {{- end }}
+          ports:
+            - containerPort: 8080
+              name: http
+              protocol: TCP
+          env:
+          {{- if .Values.connectors.env}}
+            {{ .Values.connectors.env | toYaml | nindent 12 }}
+          {{- end }}
+          {{- if eq .Values.connectors.inbound.mode "disabled" }}
+            - name: ZEEBE_CLIENT_BROKER_GATEWAY-ADDRESS
+              value: "{{ include "zeebe.names.gateway" . | replace "\"" "" }}:{{ index .Values "zeebe-gateway" "service" "gatewayPort" }}"
+            - name: ZEEBE_CLIENT_SECURITY_PLAINTEXT
+              value: "true"
+            - name: CAMUNDA_CONNECTOR_POLLING_ENABLED
+              value: "false"
+            - name: CAMUNDA_CONNECTOR_WEBHOOK_ENABLED
+              value: "false"
+            - name: SPRING_MAIN_WEB-APPLICATION-TYPE
+              value: "NONE"
+          {{- end }}
+          {{- if eq .Values.connectors.inbound.mode "credentials" }} # Not implemented yet
+            - name: ZEEBE_CLIENT_BROKER_GATEWAY-ADDRESS
+              value: "{{ include "zeebe.names.gateway" . | replace "\"" "" }}:{{ index .Values "zeebe-gateway" "service" "gatewayPort" }}"
+            - name: ZEEBE_CLIENT_SECURITY_PLAINTEXT
+              value: "true"
+            - name: CAMUNDA_CONNECTOR_POLLING_ENABLED
+              value: "false"
+            - name: CAMUNDA_CONNECTOR_WEBHOOK_ENABLED
+              value: "false"
+            - name: SPRING_MAIN_WEB-APPLICATION-TYPE
+              value: "NONE"
+          {{- end }}
+          {{- if eq .Values.connectors.inbound.mode "oauth" }} # Not implemented yet
+            - name: ZEEBE_CLIENT_BROKER_GATEWAY-ADDRESS
+              value: "{{ include "zeebe.names.gateway" . | replace "\"" "" }}:{{ index .Values "zeebe-gateway" "service" "gatewayPort" }}"
+            - name: ZEEBE_CLIENT_SECURITY_PLAINTEXT
+              value: "true"
+            - name: CAMUNDA_CONNECTOR_POLLING_ENABLED
+              value: "false"
+            - name: CAMUNDA_CONNECTOR_WEBHOOK_ENABLED
+              value: "false"
+            - name: SPRING_MAIN_WEB-APPLICATION-TYPE
+              value: "NONE"
+          {{- end }}
+          command: {{ .Values.connectors.command }}
+          resources: {{- toYaml .Values.connectors.resources | nindent 12 }}
+      {{- if .Values.connectors.podSecurityContext }}
```

Comment:
This should be after the `serviceAccountName` section.

Reply:
Good catch. Moved in the next revision.